core/src/main/resources/hudson/model/View/AsynchPeople/index.jelly

```diff
@@ -100,7 +100,7 @@ THE SOFTWARE.
       <l:progressiveRendering handler="${it}" callback="display"/>
       <table class="sortable pane bigtable" id="people">
         <tr>
-          <th />
+          <th class="minimum-width" />
```

Comment:
To improve consistency with the user list view

Reply:
Makes sense, but probably a separate PR or a changelog entry? Is it a real issue with rendering you hit? Or just housekeeping?

Reply:
> housekeeping

To improve "a bit" the consistency between the two tables :)